client/src/components/login/google-button/GoogleButton.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import { GoogleLogin } from 'react-google-login';
+import PropTypes from 'prop-types';
+import auth from '../../../auth/auth';
 
-const GoogleButton = () => {
+const GoogleButton = (props) => {
   const onSuccess = (response) => {
     // TODO: 'post' call to Server with the id_token from response
-    console.log(response); // Placeholder till server code is integrated.
+    // console.log(response); // Placeholder till server code is integrated.
+    auth.login(response.Zi.id_token, () => {
+      props.history.push('/dashboard');
```

Comment:
Check if we can use redirect here instesd of history.
https://reacttraining.com/react-router/web/api/Redirect